### PR TITLE
Update CHANGELOG.md for beta17

### PR DESCRIPTION
**Pull request by SydneyhSmith**
# PR Summary

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary.

### CODE REVIEW

**Review by PaulHigin — APPROVED** [review]
LGTM

**Comment by alerickson** [comment]
@anamnavi  just a reminder to update the release notes with the change log info when upping the version in the module manifest

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # CHANGELOG
 
+## 3.0.17-beta17
+
+### New Features
+- Add -TemporaryPath parameter to Install-PSResource, Save-PSResource, and Update-PSResource (#763)
+- Add String and SecureString as credential types in PSCredentialInfo (#764)
+- Add Install interface design (#776)
```
Comment (anamnavi) [inline CHANGELOG.md]:
```suggestion

```

Comment (anamnavi) [reply CHANGELOG.md]: this would be applicable to the next release, currently is WIP.

**Review by anamnavi — CHANGES_REQUESTED** [review]

**Review by anamnavi — APPROVED** [review]

**Pull request merged**